x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/event_filter_delete_modal.tsx

```diff
@@ -80,20 +83,40 @@ export const EventFilterDeleteModal = memo<{}>(() => {
         <EuiModalHeaderTitle>
           <FormattedMessage
             id="xpack.securitySolution.eventFilters.deletionDialog.title"
-            defaultMessage="Remove event filter"
+            defaultMessage={`Remove "{name}"`}
+            values={{
+              name: eventFilter?.name,
+            }}
           />
         </EuiModalHeaderTitle>
       </EuiModalHeader>
 
       <EuiModalBody data-test-subj="eventFilterDeleteModalBody">
         <EuiText>
-          <p>
-            <FormattedMessage
-              id="xpack.securitySolution.eventFilters.deletionDialog.mainMessage"
-              defaultMessage='You are removing event filter "{name}".'
-              values={{ name: <b className="eui-textBreakWord">{eventFilter?.name}</b> }}
-            />
-          </p>
+          <EuiCallOut
+            data-test-subj="eventFilterDeleteModalCallout"
+            title={i18n.translate(
+              'xpack.securitySolution.eventFilters.deletionDialog.calloutTitle',
+              {
+                defaultMessage: 'Warning',
+              }
+            )}
+            color="danger"
+            iconType="alert"
+          >
+            <p data-test-subj="eventFilterDeleteModalCalloutMessage">
+              <FormattedMessage
+                id="xxpack.securitySolution.eventFilters.deletionDialog.calloutMessage"
+                defaultMessage="Deleting this entry will remove it from {count} associated {count, plural, one {policy} other {policies}}."
+                values={{
+                  count: isGlobalPolicyEffected(Array.from(eventFilter?.tags || []))
+                    ? 'all'
```

Comment:
Interesting. so `count` could be a `string` (`all`) or a number and the i18n defaults the string value to plural?

Reply:
Exactly, I guess is because there is more than one char